library_collection/models.py

```diff
@@ -268,6 +268,12 @@ class Collection(models.Model):
         max_digits=5, decimal_places=2, null=True, blank=True)
     metadata_density_score_last_updated = models.DateTimeField(
         null=True, blank=True, help_text='Last time metadata density score was updated')
+    production_target_version = models.CharField(
+        max_length=255, blank=True, help_text=(
```

Comment:
s3 max length for a key is 1024 bytes, which I think is 512 UTF-8 characters, so 512 shuold prbabaly be the max_length?